drafts/w3f-#5475-coretime-marketplace-draft.md

```diff
@@ -18,23 +18,129 @@ To fully follow the material on this page, it is recommended to be familiar with
 ## Coretime
 
 <!-- Write the short summary of the coretime and the need of the Coretime Marketplace -->
-
+As polkadot is moving toward an application-focused ecosystem where all cores are a resource to be consumed and used as needed by all applications. Previously, the auction mechanism was quite a competitive process to secure a slot on parachain. With coretime rental, there is no need for slot anymore, teams can purchase instantaneous or reserve bulk coretime. This greatly decreases the barrier-to-entry for software tinkerers and parachain teams and increase the accessibility to polkadot computation without wasting valuable blockspace.<br/>
+The slot auction mechanism is not agile, creates high entry barriers, and is designed for long-running single applications (i.e., the original Polkadot vision proposed in the whitepaper). We depart from the classic lease auctions and propose an agile marketplace for coretime, where essentially coretime becomes a commodity that can be tokenized, sold, and traded. This setup maximizes the agility of Polkadot and lets the market figure out the best solution needed for applications to be successful.
+***
 ### Coretime Chain
 
 <!-- Breaking down the details of the concepts -->
 
+The Coretime Chain is a proposed new system parachain within the Polkadot Network, responsible for the sale and management of Coretime. It is designed to handle the allocation of Bulk Coretime and track ownership of Coretime as non-fungible assets (NFTs). The Coretime Chain provides information to the Relay-chain regarding the number of cores available, the tasks running on each core, and accounting information for Instantaneous Coretime Credit. Additionally, it processes renewals and allows for various manipulations of Bulk Coretime, such as transfers, partitioning, interlacing, assignment to tasks, and pooling for Instantaneous Coretime.
+
+#### Communication
+
+The Coretime Chain and Relay Chain communicate through two protocols:
+
+- **Upward Message Passing (UMP):**
+  - Coretime Chain provides Relay Chain with the following information:
+    - Number of cores to be made available.
+    - Tasks running on each core and their ratios.
+    - Accounting information for Instantaneous Coretime Credit.
+
+- **Downward Message Passing (DMP):**
+  - Relay Chain provides Coretime Chain with the following information:
+    - Number of cores available to be scheduled.
+    - Account information on Instantaneous Coretime Sales.
+
+#### Specific Functions of the Coretime Chain
```

Comment:
We don't want to include functions here

Reply:
Should be written in a way to serve different kinds of audience but still technical enough